scripts/Camera2D.gd

```diff
@@ -63,5 +63,6 @@ func _process(delta):
 	
 	# Clamp the camera's position to the desired boundaries
 	# TODO: Update boundaries
-	position.x = clamp(position.x, 0, 100)
```

Comment:
The position and zoom don't work well together. The default zoom is 0.25x with limits of (0.2-1.0x). Currently, the camera zooms to the left, possibly due to the "Anchor Mode", but when I changed the setting the point of selection doesn't match the cursor position. I'll look into this again later.